### ISSUE TITLE
Singularity instance.start clearing environtment

### ISSUE THREAD

**Issue by Shotgunosine** [apptainer/singularity#1766]
## Version of Singularity:

2.5.2-master.b258b65

### Expected behavior

Environment variables should persist into singularity instance when started with singularity instance.start.

### Actual behavior

Environmet variables are cleared. 

### Steps to reproduce behavior

This may be slightly roundabout, but here's how i've been reproducing the issue:
Write this Dockerfile:
```
FROM ubuntu:18.04

RUN useradd  --create-home --home-dir /home/testuser testuser

USER testuser
```

Singularity definition file:
```
Bootstrap: docker
Registry: http://localhost:5000
Namespace:
From: singularity_test

%startscript
    echo "Echoing $HOME" > /output/out
    echo $HOME >> /output/out
    echo "Echoing whoami:" >> /output/out
    whoami >> /output/out
    echo "Echoing env:" >> /output/out
    env >> /output/out
    echo "Echoing ls $HOME" >> /output/out    
    ls $HOME >> /output/out
    echo "Echoing ls /home/$(whoami)" >> output/out
    ls /home/$(whoami) >> /output/out
```

Then run the following commands:

```
mkdir output
mkdir shome
touch shome/this_file_is_here
docker run -d -p 5000:5000 --restart=always --name registry registry:2
docker build -t singularity_test .
docker tag singularity_test localhost:5000/singularity_test
docker push localhost:5000/singularity_test
sudo SINGULARITY_NOHTTPS=1 singularity build singularity_test.simg Singularity
singularity instance.start -B $PWD/output:/output -H $PWD/shome:/nielsond/home singularity_test.simg stest
cat output/out
```
This produces the following output:
```
Echoing 

Echoing whoami:
nielsond
Echoing env:
LD_LIBRARY_PATH=/.singularity.d/libs
PS1=Singularity> 
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
PWD=/
Echoing ls 
bin
boot
dev
environment
etc
home
lib
lib64
media
mnt
opt
output
proc
root
run
sbin
singularity
srv
sys
tmp
usr
var
Echoing ls /home/nielsond
this_file_is_here
```

If the environment variables weren't being cleared, you would expect to see something like this:
```
Echoing /home/nielsond
 /home/nielsond
Echoing whoami:
nielsond
Echoing env:
LD_LIBRARY_PATH=/.singularity.d/libs
PS1=Singularity> 
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
PWD=/
HOME=/home/nielsond
Echoing ls /home/nielsond
this_file_is_here
Echoing ls /home/nielsond
this_file_is_here
```

**Comment by jakirkham** [comment]
Should this now be closed?

**Comment by jmstover** [comment]
Hi @jakirkham 

Yes. This was fixed (or should be fixed ;) ) in PR #1800 which was merged, and is part of 2.6.0 which was released earlier tonight.

**Comment by GodloveD** [comment]
Thanks very much @jakirkham for pointing out some issues that need to be closed.  This kind of help is warmly appreciated.  Cheers! 🍺
